upup/pkg/fi/cloudup/tagbuilder.go

```diff
@@ -96,6 +96,16 @@ func buildCloudupTags(cluster *api.Cluster) (sets.String, error) {
 func buildNodeupTags(role api.InstanceGroupRole, cluster *api.Cluster, clusterTags sets.String) (sets.String, error) {
 	tags := sets.NewString()
 
+	networking := cluster.Spec.Networking
+
+	if networking == nil {
+		return nil, fmt.Errorf("Networking is not set, and should not be nil here")
```

Comment:
I'm not sure you needed to do this - it probably would have sufficed to just do

`if cluster.Spec.Networking != nil && cluster.Spec.Networking.LyftVPC != nil {`